### PR TITLE
Do spellchecking

### PR DESCRIPTION
**Pull request by cmarqu**

### CODE REVIEW

File: cocotb/drivers/__init__.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def _send_thread(self):
 
 
 class BusDriver(Driver):
-    """Wrapper around common functionality for busses which have:
+    """Wrapper around common functionality for buses which have:
```
Comment (imphil) [inline cocotb/drivers/__init__.py]: busses is valid in AE, but fine either way.

File: cocotb/share/lib/simulator/simulatormodule.c [hunk 2]
```diff
@@ -203,7 +203,7 @@ static PyObject *log_msg(PyObject *self, PyObject *args)
 }
 
 
-// Register a callback for read only state of sim
+// Register a callback for ReadOnly state of sim
```
Comment (imphil) [inline cocotb/share/lib/simulator/simulatormodule.c]: ReadOnly() is the name of our trigger. The state of the simulation is actually read-only (or readonly).

**Comment by cmarqu** [comment]
Note that this is still a draft PR currently.
I actually *am* using the amend option and I push with force, but for some reason it's not enough...

**Comment by cmarqu** [comment]
Superseded by https://github.com/cocotb/cocotb/pull/1124.

**Pull request closed without merging**